src/events/cd-event-add-child-ticket.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+  <div class="cd-child-ticket__ticket-box">
+    <div class="cd-child-ticket__header">
+       <h3 class="cd-child-ticket__ticket-name">{{ $t('Ticket')}} {{firstName|addPosession}} </h3>
```

Comment:
Posession?

Reply:
Maybe i used to wrong wording. The filter is meant to ass the child's name onto the ticket. So it would say 'Ticket - Jane' or whatever their name is.

Reply:
Then it's possession I believe :)